docs/pages/tutorials.mdx

```diff
@@ -0,0 +1,18 @@
+# MUD tutorials
+
+These tutorials teach you how to do various things with MUD.
+
+## [Modifying the getting started example](tutorials/minimal)
```

Comment:
These linked headings look kinda bad, wonder if @nickcernera has ideas for spicing them up?
https://mud-docs-1dqbb3x0k-latticexyz.vercel.app/tutorials

Reply:
I slotted yesterday for MUD community/docs stuff but need to pivot back to Sky Strife the rest of the week — can you figure something out in the meantime?

Also achal will be in the docs prettifying everything soon enough!

Reply:
Links moved.